keanu-project/src/main/java/io/improbable/keanu/templating/loop/Loop.java

```diff
@@ -123,7 +122,7 @@ public static LoopBuilder withInitialConditions(VertexDictionary initialState) {
     }
 
     private void checkIfMaxCountHasBeenReached(SequenceItem item) throws LoopDidNotTerminateException {
-        BooleanVertex stillLooping = item.get(STILL_LOOPING_LABEL);
+        BooleanVertex stillLooping = item.get(LOOP_LABEL);
```

Comment:
why did you change the name?

Reply:
no idea. I don't actually remember doing it. Will change it back